assets/src/edit-story/components/canvas/singleSelectionMovable.js

```diff
@@ -252,9 +267,15 @@ function SingleSelectionMovable({ selectedElement, targetEl, pushEvent }) {
       }}
       throttleDrag={0}
       onDragStart={({ set }) => {
+        if (isEditMode) {
+          return false;
```

Comment:
Why not just disable dragging?
When disabling dragging, Moveable dismisses any events and it's not possible to click or select text inside the edit box, even if the text edit is on top of moveable. Enabling dragging but returning `false` allows events but not dragging.